pyspecProcScripts/apod_matched_filter.py

```diff
@@ -43,23 +43,26 @@ def apod_matched_filter(s, axis='t2',
     elif convolve_method == 'lorentzian':
         convolution_set = np.exp(-abs(temp.fromaxis(axis))/sigma)
     signal_E = (abs(temp * convolution_set)**2).sum(axis)
-    signal_E /= signal_E.data.max()
+    signal_E /= abs(signal_E.data).max()
```

Comment:
on looking at this again, I realize that the data is positive to start with, so you can remove abs if you want.